helm/aws-load-balancer-controller/templates/webhook.yaml

```diff
@@ -56,6 +56,28 @@ webhooks:
     resources:
     - pods
   sideEffects: None
+- clientConfig:
```

Comment:
could we make this webhook conditional based on a configuration value. For example, `defaultControllerForService` set to `true` by default. So if you don't want to make the lbc as default controller for service of type LoadBalancer, you can install the controller with `--set defaultControllerForService=false`